.github/workflows/test.yml

```diff
@@ -91,11 +91,14 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: actions/cache@v2.1.5
+        name: Cache Cargo registry + index
         with:
           path: |
             ~/.cargo/registry
             ~/.cargo/git
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-cargo-
```

Comment:
🤔 what caches with this key?

Reply:
It's a prefix. So, previous runs that hash `Cargo.lock` will be reused and then updated in place. It saves a little time pulling mostly the same deps.

Reply:
TBH I had to fiddle with it for a bit to understand their docs on the subject.

Reply:
Aha, I see. It is certainly not clear that this is a prefix from their docs or the field name. 👍

Reply:
Maybe we could add a comment to these?